packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManagerInterface.java

```diff
@@ -76,6 +76,8 @@ public interface BaseViewManagerInterface<T extends View> {
 
   void setTransform(T view, @Nullable ReadableArray matrix);
 
+  void setBackgroundImage(T view, @Nullable ReadableArray backgroundImage);
+
```

Comment:
If we will only implement this for `<View>` for now, we probably want to keep this off of `BaseViewManager`. 

I've been working to get us to share more of the background drawing infra, eventually moving some of these to BaseViewManager, but it will take time.